website/blog/2022-01-12-announcing-oclif-v2.md

```diff
@@ -0,0 +1,109 @@
+
+oclif v2 includes many changes that improve the experience of both creating and using an oclif CLI.
+
+Going forward we do not plan to make any updates to oclif v1 and its corresponding libraries except for critical security fixes (please see the [compatibility matrix](#Compatibility-Matrix) for a list of these libraries). We are not completely dropping support yet in order to give developers time to migrate from v1 to v2 but at some point in the future we will archive v1 repositories and deprecate their versions on NPM.
+
+## What’s changing?
+
+### Repository Changes
+
+All repositories under the [oclif org](https://github.com/oclif/) now use `main` as their primary branch and `oclif-v1` as the legacy branch. The `main`  branch corresponds to oclif v2 and the `oclif-v1` branch to oclif v1. If you find any repos that don’t have a `main` branch you can safely assume that they are not used in oclif v2 and will be deprecated at some point in the future.
+
+### Consolidating Tools & Libraries
+
+The [`oclif-dev`](https://github.com/oclif/dev-cli) CLI has now been incorporated into [`oclif`](https://github.com/oclif/oclif). Now you no longer need to install a separate package to manage your entire CLI’s lifecycle.
+
+The following libraries have been consolidated under [`@oclif/core`](https://github.com/oclif/core) and will be deprecated at some point in the future. You can read the [migration guide](https://github.com/oclif/core/blob/main/MIGRATION.md) to see how to update your CLI or plugin to use the new core library.
+
+* `@oclif/command`
+* `@oclif/config`
+* `@oclif/error`
+* `@oclif/help`
+* `@oclif/parser`
+
+### No more single vs multi command CLIs
+
+There is only 1 "type" of oclif v2 CLI but it can have as few or as many commands as a developer wants. This also means there’s only one command for generating CLIs, `oclif generate` as opposed to the old `oclif single` and `oclif multi` commands.
+
+### Node > 12
+
+In order to ensure oclif CLIs are as secure as possible all v2 libraries and plugins only support Node 12 or higher as of now and, going forward, will only be supporting Node Maintenance or higher as defined by Node's release schedule here: https://nodejs.org/en/about/releases/.
+
+## What’s new?
+
+### New example CLI
+
+[`oclif-hello-world`](https://github.com/oclif/hello-world/) is our new example repo. This is also the repo that is [used as a template](https://github.com/oclif/oclif/blob/edc6616e51/src/generators/cli.ts#L74) when running `oclif generate` to create a new CLI. This repo will always be able to be referenced as an example of what an oclif v2 CLI should look like.
+
+### Spaced commands
+
+oclif CLIs can now use spaces to separate topics and subcommands instead of colons. To enable this feature simply add `“topicSeparator": " "` to the oclif config in your package.json. You can see an example of this in our [example repo](https://github.com/oclif/hello-world/blob/main/package.json#L55).
```

Comment:
```suggestion
oclif CLIs can now use spaces, instead of colons, to separate topics and subcommands. To enable this feature, simply add `“topicSeparator": " "` to the oclif config in your package.json. See an example in our [example repo](https://github.com/oclif/hello-world/blob/main/package.json#L55).
```